Buildings/Experimental/DHC/EnergyTransferStations/BaseClasses/Validation/CollectorDistributor.mo

```diff
@@ -0,0 +1,715 @@
+  connect(senT2_2Ret2.T,conTChiWat6.u_m)
+    annotation (Line(points={{60,-389},{60,-380},{100,-380},{100,-332}},color={0,0,127}));
+  connect(vol3.ports[1],sou2_4.port_a)
+    annotation (Line(points={{217,-400},{190,-400}},color={0,127,255}));
+  connect(senMasFlo2_6.port_b,vol3.ports[2])
+    annotation (Line(points={{150,-360},{221,-360},{221,-400}},color={0,127,255}));
+  connect(sou2_4.port_b,coo5.port_a)
+    annotation (Line(points={{170,-400},{100,-400}},color={0,127,255}));
+  connect(senT1Ret2.port_b,bou3.ports[1])
+    annotation (Line(points={{-110,-260},{-180,-260},{-180,-238}},color={0,127,255}));
+  connect(sou2_3.port_b,coo4.port_a)
+    annotation (Line(points={{170,-260},{100,-260}},color={0,127,255}));
+  connect(m2_6.y,sou2_4.m_flow_in)
+    annotation (Line(points={{198,-320},{180,-320},{180,-388}},color={0,0,127}));
+  connect(colDis2.ports_bCon[1],senT2_1Sup2.port_a)
+    annotation (Line(points={{-48,-230},{-40,-230},{-40,-200},{44,-200},{44,-220},{50,-220}},color={0,127,255}));
+  connect(colDis2.ports_bCon[2],senT2_2Sup2.port_a)
+    annotation (Line(points={{-56,-230},{-56,-204},{40,-204},{40,-360},{50,-360}},color={0,127,255}));
+  connect(senT2_1Ret2.port_b,colDis2.ports_aCon[1])
+    annotation (Line(points={{50,-260},{24,-260},{24,-220},{-14,-220},{-14,-230},{-24,-230}},color={0,127,255}));
+  connect(senT2_2Ret2.port_b,colDis2.ports_aCon[2])
+    annotation (Line(points={{50,-400},{20,-400},{20,-224},{-32,-224},{-32,-230}},color={0,127,255}));
+  connect(bou3.ports[2],senMasFlo3.port_a)
+    annotation (Line(points={{-180,-242},{-180,-220},{-140,-220}},color={0,127,255}));
+  connect(senT1Sup2.port_b,colDis2.port_aDisSup)
+    annotation (Line(points={{-90,-220},{-80,-220},{-80,-240},{-60,-240}},color={0,127,255}));
+  connect(senT1Ret2.port_a,colDis2.port_bDisRet)
+    annotation (Line(points={{-90,-260},{-80,-260},{-80,-246},{-60,-246}},color={0,127,255}));
+  annotation (
+    Documentation(
+      info="<html>
+<p>
+This model validates
+<a href=\"modelica://Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.CollectorDistributor\">
+Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.CollectorDistributor</a>
+in a configuration where the model is used to ensure hydronic decoupling between
+</p>
+<ol>
+<li>
+a primary circuit which mass flow rate varies from 0 to 1.1 times
+<code>m_flow_nominal</code>,
+</li>
+<li>
+a first secondary circuit which mass flow rate varies from 0.5 to 1 times
+<code>m_flow_nominal</code>,
+</li>
+<li>
+a second secondary circuit which flow rate is constant, equal to 0.5 times
+<code>m_flow_nominal</code>.
+</li>
+</ol>
+</html>",
+      revisions="<html>
+<ul>
+<li>
+July 31, 2020, by Antoine Gautier:<br/>
+First implementation
+</li>
+</ul>
+</html>"),
+    Diagram(
+      coordinateSystem(
+        preserveAspectRatio=false,
+        extent={{-260,-560},{260,440}})),
```

Comment:
change the extent to `{{-260,-440},{260,440}}`

Reply:
Addressed.
I also updated the documentation as it was obsolete.